eng/service.proj

```diff
@@ -20,7 +20,7 @@
     <PerfProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\perf\**\*.csproj" />
     <StressProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\stress\**\*.csproj" />
     <SampleApplications Include="..\samples\**\*.csproj" />
-    <SrcProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\*.csproj" Exclude="@(TestProjects);@(SamplesProjects)"/>
+    <SrcProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\src\**\*.csproj" />
```

Comment:
@weshaggard this was always wrong, but looking for anything that uses it or sets it to anything other than `false` turned up nothing. I found it because I was trying to build *just* source projects to get the analyzers to run while not taking forever on my little laptop building perf, stress, and tests. I can't see any harm in this fix.

Reply:
Alternatively, if this is meant to support older projects (but even track 1 seems to use "src"), it should probably be:

```suggestion
    <SrcProjects Include="..\sdk\$(ServiceDirectory)\$(Project)\**\*.csproj" Exclude="@(TestProjects);@(SamplesProjects);@(PerfProjects);@(StressProjects)" />
```

From running `dir sdk\*\*.csproj` in PowerShell from the root, `SrcProjects` in main doesn't resolve to anything.

Reply:
You are probably correct that "src" would probably work for most places. I'm not sure that there are directories with other folder types that we intentionally want to build. I'm fine with your proposed fix and we can adjust if we find any issues.

Reply:
Which proposed fix? As it is now? There are no files that match the old pattern anyway, and the `Exclude` attribute is unnecessary in that case because the paths do not intersect.

Reply:
As it is with this PR currently would also be faster to compute.